packages/template-tag-codemod/src/index.ts

```diff
@@ -101,11 +106,15 @@ export async function ensureAppSetup() {
   try {
     pkg = resolverLoader.resolver.packageCache.get(process.cwd());
   } catch (err) {
-    console.error(`Run template-tag-codemod inside a Ember app.`);
+    console.error(`Run template-tag-codemod inside a Ember app or run an Embroider build of the app first`);
```

Comment:
isn't the "embroider prebuild" the "Embroider build" you're talking about here? I feel like this could error for a million different reasons and maybe we should actually share `err.message` or something?